### DISCUSSION TITLE
Installation Issue / Installationsfehler

### DISCUSSION THREAD

**Discussion by LufthansaPilot11** [flybywiresim/aircraft#4659] [Help] [answered]
Hello,

I have problems with installing the A32NX Mod. I have installed MSFS2020 via the Microsoft Store and followed the instructions for the manuel installation. As I downloaded the latest version it contains an folder called "flybywire-aircraft-a320-neo" although it should include an folder called "A32NX" which is stated in the installation instructions. This so-called folder "A32NX" should be copied into the community folder. As there is only the folder "flybywire-aircraft-a320-neo" I copied this one in the Community, but with no success. I have also tried to install it with the Installer but this also did not work for me. What exactly am I doing wrong ?

**German:**
Hallo,

Ich habe Probleme mit der Installation des A32NX Mod. Ich habe MSFS2020 über den Microsoft Store installiert und die Anweisungen für die manuelle Installation befolgt. Als ich die neueste Version heruntergeladen habe, enthält sie einen Ordner namens "flybywire-plane-a320-neo", obwohl sie einen Ordner namens "A32NX" enthalten sollte, der in der Installationsanleitung angegeben ist. Dieser sogenannte Ordner "A32NX" sollte in den Community-Ordner kopiert werden. Da es nur den Ordner "flybywire-plane-a320-neo" gibt, habe ich diesen in der Community kopiert, aber ohne Erfolg. Ich habe auch versucht, es mit dem Installer zu installieren, aber das hat auch bei mir nicht funktioniert. Was genau mache ich falsch ?

![Screenshot (12)](https://user-images.githubusercontent.com/83464882/116721274-6b6b6900-a9dd-11eb-913a-92e149e167f2.png)

**Comment by Kamegohan** [comment]
There seems nothing to be wrong. Have the same folder in my community folder and works perfect for me. Have you selected the A320neo by flybywire (LEAP) in the plane selection? It is a separate plane now which has to be chosen manually. I am not sure if you have to install it from the build-in store in MSFS (no worry, it's free). 
Please double-check you have not selected the pre-installed A320neo from Asobo

**Comment by ExampleWasTaken** [comment, marked as answer]
@LufthansaPilot11 the A32NX folder no longer exists. The folder that needs to go into the community folder is now called `flybywire-aircraft-a320-neo`.
Could you point out the location where it mentions the A32NX folder so we can update it.

**Reply by LufthansaPilot11** [reply]
"A32NX" is mentioned on this page "https://github.com/flybywiresim/a32nx", in the installation instructions at the bottom.

**Comment by LufthansaPilot11** [comment]
Hello,

I found out that I installed it correctly each time and the mod works for me now. I always selected the default Airbus and did not know that there is now a second Aircraft to choose.

**Reply by ExampleWasTaken** [reply]
In order to receive information like that, I recommend joining the [official Discord](https://discord.gg/flybywire) or follow FlyByWire on [Twitter](https://twitter.com/FlyByWireSim).
